src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/ReferenceConfidenceModel.java

```diff
@@ -217,6 +217,11 @@ public ReferenceConfidenceModel(final SampleList samples,
             }
         }
 
+        // Ensuring that we remove any indel informativeness data we may have attached to the underlying reads for caching purposes
```

Comment:
Add an additional comment explaining briefly why it is necessary to clear the cached data.